libs/moment/include/moment/clock_interfaces.hpp

```diff
@@ -30,8 +30,10 @@ class ClockInterface
 {
 public:
   using ChronoClock = std::chrono::steady_clock;
+  using SystemClock = std::chrono::system_clock;
```

Comment:
Turns out doing it in this way does not work well - one way around this might be to template the clock interface on steady vs. system clock although the solution might look ugly.